release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -72,6 +72,27 @@ For more information, see xref:../installing/installing_aws/installing-aws-vpc.a
 [id="ocp-4-14-web-console"]
 === Web console
 
+[id="ocp-4-14-administrator-perspective"]
+==== Administrator Perspective
+
+With this release, there are several updates to the *Administrator* perspective of the web console. You can:
+
+* Narrow down the list of resources in a list view or search page with exact search capabilities. This will help when you have similarly named resources and fuzzy search doesn't narrow down your search.
```

Comment:
Just a question: is fuzzy search a legit term that's used in the industry and/or our docs? Disregard if it is.

Reply:
Yes, it is correct only in this case.

> It is only correct to use "fuzzy" as an adjective when referring to "fuzzy searches"

From SSG https://redhat-documentation.github.io/supplementary-style-guide/#_f